### ISSUE TITLE
Upload function not working

### ISSUE THREAD

**Issue by Darktortue** [HavocFramework/Havoc#145]
Hey I'm having some issue when uploading files.

I'm trying to upload files on a W10 machine but I always face the errors :
```
[+] Send Task to Agent [118 bytes]
[!] Win32 Error: ERROR_INVALID_NAME [123]
[!] Failed to parse FS::Upload response
```
![uploadbug](https://user-images.githubusercontent.com/11302315/194528294-bf69a8fc-e45d-4785-9946-6d339f602021.PNG)

I'm 100% sure that the paths are correct and that I have write permission on the target folder.

Here's the output from debug mode of teamserver :
![uploadbug1](https://user-images.githubusercontent.com/11302315/194528496-b0ad1915-4c42-4cec-884b-8a0919d380e9.PNG)

**Comment by Cracked5pider** [comment]
Alright, thank you so much for reporting it and providing enough info. going to take a look and fix it.

**Comment by Cracked5pider** [comment]
Fixed in commit: [b45921f](https://github.com/HavocFramework/Havoc/commit/b45921f487d0e9746676f810292453b762a60d2f)
![image](https://user-images.githubusercontent.com/51360176/194785537-eac4f782-5b22-4e83-a68a-36c32919f911.png)
